packages/airnode-node/src/handlers/call-api.ts

```diff
@@ -55,17 +54,19 @@ function buildOptions(
   };
 }
 
-function createPrintableValue(value: adapter.ValueType): string {
-  if (Array.isArray(value)) return JSON.stringify(value);
```

Comment:
This was definitely done this way for a reason but I don't remember why. Are you sure that stringifying everything is ok?

Reply:
I've changed the response to be a JSON, but I've confirmed that it works (I deployed Airnode and used the testing gateway). I will document it's output as part of https://api3dao.atlassian.net/browse/AN-355

Reply:
Btw. the response can now be a really big JSON, because the `rawValue` will be whatever the API response is - but I think it's a worthy information to have and users can just copy the JSON to any online format tool to better visualize what's in there.

And I plan to add all of that to the docs tomorrow.